src/devtools/client/inspector/components/App.tsx

```diff
@@ -102,7 +103,7 @@ const InspectorApp: FC = () => {
                   <div className="devtools-sidebar-tabs">
                     <div className="tabs">
                       <nav className="tabs-navigation">
-                        <ul className="tabs-menu" role="tablist">
+                        <ResponsiveTabs activeIdx={availableTabs.indexOf(activeTab)}>
```

Comment:
Since we're not using `<ul>` for the container anymore, the items should not be `<li>`s anymore either.